cmd/girafSimulate/girafSimulate_test.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"github.com/vertgenlab/gonomics/dna"
+	"github.com/vertgenlab/gonomics/dnaTwoBit"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/genomeGraph"
+	"github.com/vertgenlab/gonomics/giraf"
+	"io/ioutil"
+	"testing"
+)
+
+func TestGirafSimulate(t *testing.T) {
+	graph := MakeTestGraph()
+	out, err := ioutil.TempFile("", "GirafSimulate_test")
+	exception.PanicOnErr(err)
+	out.Close()
```

Comment:
Does this return an error?

Reply:
Only error is 'file already closed'. So checking is unnecessary IMO.